### PR TITLE
Update CNV doc attributes

### PR DESCRIPTION
**Pull request by vikram-redhat**
@ousleyp @aburdenthehand @bgaydosrh - FYI.

### CODE REVIEW

**Review by bgaydosrh — APPROVED** [review]
Looks good to me. :+1: 

Bob

**Review by ousleyp — CHANGES_REQUESTED** [review]
For the most part, this LGTM, but there were a few places where I think we should use `{CNVProductNameStart}` instead of `{CNVProductName}`.

**Comment by vikram-redhat** [comment]
Great catches @ousleyp! Now only if @aburdenthehand would give us the ack as well ....

File: cnv/cnv_install/cnv-about-cnv.adoc [hunk 1]
```diff
@@ -4,11 +4,11 @@ include::modules/cnv-document-attributes.adoc[]
 :context: cnv-about-cnv
 toc::[]
 
-Learn about {ProductName}'s capabilities and support scope.
+Learn about {CNVProductName}'s capabilities and support scope.
 
 include::modules/cnv-what-you-can-do-with-cnv.adoc[leveloffset=+1]
 
-== {ProductName} support
+== {CNVProductName} support
```
Comment (aburdenthehand) [inline cnv/cnv_install/cnv-about-cnv.adoc]: Starts a title - should this be `{CNVProductNameStart}`?

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 2]
```diff
@@ -53,9 +53,9 @@ cannot connect to the requested virtual machine. On the client, update the
...
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1693532[*BZ#1693532*])
 
-* {ProductName} cannot reliably identify node drains that are triggered by
+* {CNVProductNameStart} cannot reliably identify node drains that are triggered by
 running either `oc adm drain` or `kubectl drain`. Do not run these commands on
-the nodes of any clusters where {ProductName} is deployed. The nodes might not
+the nodes of any clusters where {CNVProductName} is deployed. The nodes might not
 drain if there are virtual machines running on top of them.
 The current solution is to put nodes into maintenance.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1707427[*BZ#1707427*])
```
Comment (aburdenthehand) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: Not an issue with this line, but on line 69 there is an instance of `CNV` that should be replaced with `{CNVProductNameStart}`

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 3]
```diff
@@ -5,20 +5,20 @@ include::modules/cnv-document-attributes.adoc[]
 toc::[]
 
 
-== About {ProductName}
+== About {CNVProductName}
 
 include::modules/cnv-what-you-can-do-with-cnv.adoc[leveloffset=+2]
 
-=== {ProductName} support
+=== {CNVProductName} support
```
Comment (aburdenthehand) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: Starts a title - should this be `{CNVProductNameStart}`?

File: modules/cnv-networking-glossary.adoc [hunk 4]
```diff
@@ -4,15 +4,15 @@
 // * cnv/cnv_users_guide/cnv-configuring-pxe-booting.adoc
 
 [id="cnv-networking-glossary_{context}"]
-= {ProductName} networking glossary
+= {CNVProductName} networking glossary
```
Comment (aburdenthehand) [inline modules/cnv-networking-glossary.adoc]: Starts a title - should this be `{CNVProductNameStart}`?

**Comment by aburdenthehand** [comment]
@vikram-redhat - there's a few instances of ProductName starting a title, which I think should be capitalised, and one instance (release notes) of `CNV`.
Otherwise it's a go.

**Comment by vikram-redhat** [comment]
Thanks @aburdenthehand. All fixed, and found a few more. 

Merging and CPing this.

**Pull request merged**